subgraph/bin/thegraph

```diff
@@ -17,51 +18,58 @@ const executeCommand = async (command) => {
       console.log(`stderr: ${stderr}`)
     }
     console.log('stdout:', stdout)
-    console.log('stderr:', stderr)
   } catch (e) {
     // should contain code (exit code) and signal (that caused the termination).
     console.error(`exec error: ${e}`)
   }
 }
 
+const checkAccessToken = () => {
+  if (!process.env.SUBGRAPH_DEPLOY_KEY) {
+    throw new Error(
+      `
+      Missing access token in SUBGRAPH_DEPLOY_KEY env.
+      You can get a token from https://thegraph.com/hosted-service/dashboard?account=unlock-protocol
+      `
+    )
+  }
+}
+
 const codegen = async () => {
-  console.log('yarn codegen')
+  console.log('codegen...')
   await executeCommand('yarn codegen')
 }
 
 // Creating subgraphs is only available from hosted-service dashboard
 // yarn graph create $network_name-v2 --node https://api.thegraph.com/deploy/ --access-token $SUBGRAPH_DEPLOY_KEY"
 const build = async (network) => {
-  console.log(`Building subgraph for ${network}`)
-  executeCommand(`yarn graph build --network ${network}`)
+  console.log(`Building subgraph for ${network} (${networkName(network)})`)
+  await executeCommand(`yarn graph build --network ${networkName(network)}`)
 }
 
 const deploy = async (network, graphLabel) => {
-  if (!process.env.SUBGRAPH_DEPLOY_KEY) {
-    throw new Error(
-      'Missing --access-token from https://thegraph.com/hosted-service/dashboard?account=unlock-protocol'
-    )
-  }
-
   if (!network) {
     throw new Error('Missing network name')
   }
 
   const graphNode = 'https://api.thegraph.com/deploy/'
   const ipfs = 'https://api.thegraph.com/ipfs/'
-  const subgraphName = `unlock-protocol/${network}-v2`
+
+  const subgraphName = `unlock-protocol/${
```

Comment:
same here? We would add `subgraphName` ?

Reply:
that can actually be parsed from the endpoint